modules/apps/27-interchain-accounts/ibc_module_test.go

```diff
@@ -45,7 +45,20 @@ func (suite *InterchainAccountsTestSuite) SetupTest() {
 	suite.coordinator = ibctesting.NewCoordinator(suite.T(), 3)
 	suite.chainA = suite.coordinator.GetChain(ibctesting.GetChainID(0))
 	suite.chainB = suite.coordinator.GetChain(ibctesting.GetChainID(1))
-	suite.chainC = suite.coordinator.GetChain(ibctesting.GetChainID(2))
+
+	// modify mock module (ica authentication module) to no-op on OpenChanInit
+	// instead of attempting to claim channel capability
+	mockModuleA := suite.chainA.GetSimApp().GetMockModule()
+	mockModuleB := suite.chainB.GetSimApp().GetMockModule()
+	onChanOpenInit := func(ctx sdk.Context, order channeltypes.Order, connectionHops []string,
+		portID, channelID string, chanCap *capabilitytypes.Capability,
+		counterparty channeltypes.Counterparty, version string,
+	) error {
+		// do not claim channel capability
+		return nil
+	}
```

Comment:
I'm not sure what the desired functionality is here (for OnChanOpenInit)

Should the controller chain code (the ICA module) claim the channel capability or let the authentication module do that? Should the controller chain code issue an app capability? @AdityaSripal

Reply:
Since the ICA-Auth is sitting underneath ICA. i.e. ICA-Auth is the base application. You don't even need to claim the capability imo. Just pass it through to the underlying IBC-Auth.

They're both sharing the same scopedKeeper anyway